static/js/components/GroupSources.jsx

```diff
@@ -303,17 +399,23 @@ const GroupSources = ({ route }) => {
 
   const data = sources.map((source) => [
     source.id,
+    source.alias,
     source.ra,
     source.dec,
+    ra_to_hours(source.ra),
+    dec_to_dms(source.dec),
     source.redshift,
-    source.classifications,
-    source.groups,
-    source.recent_comments,
+    source.latest_class,
+    source.groups.map((group) => {
+      return group.name;
+    }),
+    source.saved_at,
+    "", // this last one is for the finder chart
```

Comment:
Can you please add a note explaining why an empty string is required here for the finder chart?